Sources/SourceKitLSPAPI/BuildDescription.swift

```diff
@@ -72,6 +81,18 @@ private struct WrappedClangTargetBuildDescription: BuildTarget {
         return description.clangTarget.headers.map(\.asURL)
     }
 
+    var resources: [URL] {
+        return description.resources.map { URL(fileURLWithPath: $0.path.pathString) }
+    }
```

Comment:
Shouldn’t this use `asURL` instead of creating a URL from the path string? (I had too many issues with file path conversions recently, so I think this conversion should be somewhere centralized.)